e2e/settings.spec.ts

```diff
@@ -52,6 +51,20 @@ const legacyCertifications = [
   ]
 ];
 
+const portfolio = {
+  title: 'foo bar',
+  url: 'https://foo.bar',
+  image: 'https://foo.bar/image.png',
+  description: 'foo bar'
+};
+
+const personalInformation = {
+  name: 'Full Stack User',
+  location: '',
+  picture: '',
+  about: ''
+};
```

Comment:
I'm not 100% sure what you mean by "inline". Do you mean something like `page.getByLabel('Some location')` instead of `page.getByLabel(personalInformation.location)`?